Cookbook/Technical-Documents/How to use Slack?.md

```diff
@@ -0,0 +1,146 @@
+# How to use Slack?
+
+In Babylon we try to be as much remote friendly as we can.
+Slack is a tool we rely on a lot to help us communicate with everyone without having to be physically present.
+On the first day at Babylon, one of the first things you have automatically access is to your Slack account.
+Although this is a pretty straightforward instant messaging tool, there a few rules and tips everyone should be aware.
+
+## Which channels to join?
+
+### #ios
+The public channel for any babylonian to reach to the iOS Team. This channel should be regularly checked by the iOS support engineers.
+
+### #ios-underground
+The private channel for the iOS Team. It's the place to raise problems, have discussions, make the team aware of your whereabouts or just to say "Good morning 👋".
+
+### #ios-meeting-outcomes
+The private channel to post the outcomes from the iOS team meetings.
+Each message represents a meeting and should have the following format:
+
+```
+**<title>, <date>**
+<what was discussed>
+Recording: <recording link>
+```
+Example:
+
+```
+**Retro, 11th Nov 2019**
+Board: https://trello.com/b/8OC8ahZz/retro-11th-nov-2019
+Actions: For @catarina to change the PR Party documentation to address the changes in the process that were discussed.
+Recording: https://babylonhealth.zoom.us/recording/share/2q99Zb7Oxg0Kk7yIAVITuGUZVCs2_eV7Vge2Ppmjq3WwIumekTziMw
+```
+
+### #ios-pedia
+A private channel used to make announcements or to open discussions on broader topics within the iOS Team.
+Each announcement or discussion should be just one message and the discussion should happen within that message thread.
+To understand the status of a message we tag the messages following:
```

Comment:
This tag should be in the message body, not reaction as anyone can add reactions and it can get lost